src/method/method_callback.cpp

```diff
@@ -230,6 +233,30 @@ namespace cubmethod
     return error;
   }
 
+  int
+  callback_handler::make_out_resultset (packing_unpacker &unpacker)
+  {
+    uint64_t query_id;
+    unpacker.unpack_all (query_id);
+
+    cubmethod::query_handler *query_handler = get_query_handler_by_qid (query_id);
+    cubmethod::query_result *qresult = query_handler->get_current_result();
+
+    make_outresult_info info;
+
+    query_handler->set_prepare_column_list_info (info.column_infos, *qresult);
+    query_handler->set_qresult_info (info.qresult_infos);
+
```

Comment:
necessary to check "query_handler is null"

Reply:
I've fixed